### ISSUE TITLE
[Feature Request] Add verbose parameter to LR schedules

### ISSUE THREAD

**Issue by ayhmdalila** [tensorflow/tensorflow#115917]
### Issue type

Feature Request

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

source

### TensorFlow version

2.20.0

### Custom code

Yes

### OS platform and distribution

WSL Ubuntu

### Current behavior?

Requesting that verbose parameter be added to TF's learning rate schedulers (such as in tf.keras.optimizers.schedules.) in order to print the learning rate per epoch/step.. etc.

### Standalone code to reproduce the issue

```shell
None
```

### Relevant log output

```shell

```
